web/src/postprocess_content.ts

```diff
@@ -166,35 +167,26 @@ export function postprocess_content(html: string): string {
                     inline_image.classList.add("landscape-thumbnail");
                 }
             }
-        } else {
```

Comment:
Why did you pull this out of the `else` clause? I think this may result in potentially both blocks running for images; is that intended?